### PR TITLE
feat: add unsafe-password support to cast wallet import

### PR DESCRIPTION
**Pull request by misko9**
Adds an `--unsafe-password` option to the `cast wallet import` cli.

## Motivation

Our test suite uses the foundry docker image to start `anvil` and then executes `cast` commands using that docker image. As test cases have been built, we found that it would be beneficial to be able to import an account using `cast wallet import`, but it requires a password prompt making this difficult in our test environment while other commands such as `cast wallet new` do not have this limitation.

## Solution

An option, `--unsafe-password`, was added to the `cast wallet import` command to support bypassing the password prompt by passing the password in the cli. This solution is similar to the `cast wallet new` command along with other `cast` commands that use a keystore.

Note: this new option does not currently include a test case. There are no existing test cases for `cast wallet import` to mirror. I have a test case prepped, but subsequent test runs with it will fail due to the account name's keystore already existing unlike `cast wallet new` which will have a unique keystore filename on each run.

### CODE REVIEW

File: crates/cast/bin/cmd/wallet/mod.rs [hunk 1]
```diff
@@ -135,6 +135,10 @@ pub enum WalletSubcommands {
         /// (~/.foundry/keystores)
         #[clap(long, short)]
         keystore_dir: Option<String>,
+        /// Password for the JSON keystore in cleartext
+        /// This is unsafe, we recommend using the default hidden password prompt
+        #[clap(long, env = "CAST_PASSWORD", value_name = "PASSWORD")]
```
Comment (Evalir) [inline crates/cast/bin/cmd/wallet/mod.rs:140]: should this maybe be `CAST_UNSAFE_PASSWORD`?

**Review by Evalir — CHANGES_REQUESTED** [review]

File: crates/cast/bin/cmd/wallet/mod.rs (hunk 1 above)
Comment (misko9) [reply crates/cast/bin/cmd/wallet/mod.rs:140]: `CAST_PASSWORD` was chosen to have consistency with `cast wallet new`, but I'm more than happy to make it `CAST_UNSAFE_PASSWORD` too.

**Review by Evalir — APPROVED** [review]
pending @mattsse / @klkvr

**Review by klkvr — APPROVED** [review]
lgtm

**Comment by Evalir** [comment]
ci failure unrelated

**Pull request merged**